lib/controllers/orders.js

```diff
@@ -27,9 +27,32 @@ module.exports = Router()
       const order = await Order.getById(id);
 
       res.send(order);
-    } catch (err) {
+    } catch(err) {
       next(err);
     }
   })
-  .put('/:id', async (req, res, next) => {})
-  .delete('/:id', async (req, res, next) => {});
+  .put('/:id', async (req, res, next) => {
+    try {
+      const id = req.params.id;
+      const quantity = req.params.quantity;
+      const order = await Order.updateById(id, quantity);
```

Comment:
This is where you would be able to actually implement the methods you added to `OrderService`, same as the example in the `.post` route. You can use `OrderService.updateOrder` here, which will send the text message and then make a call to your `Order` model's updateById method. (Same with the delete)